package.json

```diff
@@ -3,7 +3,14 @@
   "bugs": {
     "url": "https://github.com/iopipe/serverless-plugin-iopipe/issues"
   },
-  "dependencies": {},
+  "dependencies": {
+    "cosmiconfig": "^3",
+    "debug": "^2.6.8",
+    "fs-extra": "^5.0.0",
+    "lodash": "^4.17.4",
```

Comment:
fwiw, it looks like you're only using https://www.npmjs.com/package/lodash.find lodash.find in the source at the moment.